service/awsconfig/v7/key/key.go

```diff
@@ -108,11 +111,18 @@ func ClusterNamespace(customObject v1alpha1.AWSConfig) string {
 	return ClusterID(customObject)
 }
 
+// ClusterOrganization returns the org name from the custom object.
+// It uses ClusterCustomer until this field is renamed in the custom object.
+func ClusterOrganization(customObject v1alpha1.AWSConfig) string {
+	return ClusterCustomer(customObject)
+}
+
 func ClusterTags(customObject v1alpha1.AWSConfig) map[string]string {
 	cloudProviderTag := ClusterCloudProviderTag(customObject)
 	tags := map[string]string{
-		cloudProviderTag: CloudProviderTagOwnedValue,
-		ClusterTagName:   ClusterID(customObject),
+		cloudProviderTag:    CloudProviderTagOwnedValue,
```

Comment:
Why is this lower case?

Reply:
It's a variable. We need to call ClusterCloudProviderTag because the tag name includes the clusterID.

Reply:
Yeah, just wondering, the other vars are upper, but this is lower. #consistency

Reply:
Cool good to check. The others are constants because their tag names are static.